lib/dotcom_web/templates/stop/index.html.eex

```diff
@@ -1,9 +1,8 @@
-<% track_template() %>
 <% subway? = @mode == :subway%>
 <% h_level_route = if @mode == :commuter_rail, do: :h3, else: :h2 %>
 <div class="container">
   <div class="page-section">
-    <h1>MBTA Stations</h1>
+    <h1>{~t(MBTA Stations)}</h1>
     <%= render "_search_bar.html" %>
     <%= mode_tab_list %ModeTabList{
       links: for mode <- [:subway, "commuter-rail", :ferry] do
```

Comment:
Please translate the "Explore stations by mode" label (a few lines below this one)